buildroot/README.md

```diff
@@ -0,0 +1,11 @@
+# Helpers for Buildroot 
+
+These scripts allow you to build this package outside of the main buildroot tree. This makes debugging easier.
+
+# Usage
+
+```
+./cmake.sh
+make
+./install.sh
+```
```

Comment:
It would be nice to say where the project needs to be pasted relative to the Buildroot directory? Just inside it?

Reply:
Especially because the `conf.sh` file hardcodes this.